charter.html

```diff
@@ -92,8 +92,15 @@ <h1 id="title">[DRAFT] Distributed Tracing Group Charter</h1>
 
     <p class="mission">The
       <strong>mission</strong> of the
-      <a href="">Distributed Tracing Working Group</a> is to define standards for interoperability between tracing tools. Interoperability
-      refers to:
+      <a href="">Distributed Tracing Working Group</a> is to define standards for interoperability between tracing tools. A tracing tool refers to a tool which is either
```

Comment:
thinking about it I feel we may extend the charter to platforms as well. So it will include ASP.NET as a platform or service meshes/proxies to be trace context aware